pkg/release/push.go

```diff
@@ -376,14 +377,13 @@ func (p *PushBuild) copyStageFiles(stageDir string, files []stageFile) error {
 // PushReleaseArtifacts can be used to push local artifacts from the `srcPath`
 // to the remote `gcsPath`. The Bucket has to be set via the `Bucket` option.
 func (p *PushBuild) PushReleaseArtifacts(srcPath, gcsPath string) error {
-	logrus.Info("Pushing release artifacts")
+	dstPath := gcs.NormalizeGCSPath(filepath.Join(p.opts.Bucket, gcsPath))
+	logrus.Infof("Pushing release artifacts from %s to %s", srcPath, dstPath)
 
-	copyOpts := gcs.DefaultGCSCopyOptions
-	copyOpts.NoClobber = pointer.BoolPtr(p.opts.AllowDup)
-
-	return errors.Wrap(gcs.CopyToGCS(
-		srcPath, filepath.Join(p.opts.Bucket, gcsPath), copyOpts,
-	), "copy artifacts to GCS")
+	return errors.Wrap(
+		gcp.GSUtil("-m", "rsync", "-r", srcPath, dstPath),
```

Comment:
@saschagrunert -- for a follow-up, can we see if we can weave an rsync/bucket-to-bucket copy option into `pkg/gcp/gcs/gcs.go` instead?

Reply:
Yes, I can work on that :+1: